.github/workflows/java_ci.yml

```diff
@@ -0,0 +1,59 @@
+name: Java CI with Gradle
+
+on:
+  push:
+    branches: [github-ci-for-java]
```

Comment:
If we use it we need to change to master

Reply:
Okay, let us change this in the next step.